OpenRA.Mods.Common/UtilityCommands/LegacyMapImporter.cs

```diff
@@ -180,8 +185,10 @@ public void ConvertIniMap(string iniFile)
 			LoadActors(file, "INFANTRY");
 			LoadSmudges(file, "SMUDGE");
 
+			mapPlayers = new MapPlayers();
 			foreach (var p in players)
 				LoadPlayer(file, p, legacyMapFormat == IniMapFormat.RedAlert);
+			map.PlayerDefinitions = mapPlayers.ToMiniYaml().ToList();
```

Comment:
If you swap the order of waypoint and player parsing you could add MakeDefaultPlayers here and then drop the other block  completely.